Wendy/assignment1Answers.html

```diff
@@ -0,0 +1,219 @@
+					<h3>a) "1" == 1  true.</h3>
+					<p>The == equality operator says "kind of the same" and they are.  Javascript reads the equality operator from right to left and changes the operators to the same type if needed.  
+
+					<h3>b) "1" === 1 false.</h3>
+					<p>The "===" is "exactly equal to"  No room for kind of.</p>
+
+					<h3>c) x == 'x'  false.</h3>
+					<p>  If x is defined as a string, the 'x' is saying string 'x' and if your string is for example var x = 'snowman', 'snowman' != 'x'.  If x is a number, it is still false because a number is not a string.</p>
+
+
+					<h3>d) x == (x+'') false.</h3>
+					<p>This is false because the '' empty quotes make things falsy!</p>
+
+					<h3>e) '' == ' ' false.</h3>  Nothing is not equal to a space.  A space is something.</h3>
+
+					<h3>f) x = true  true.</h3>
+					<p>  Assigning the boolean value of true to x.</p>
+				</div>
+
+				<div class="col-md-4">
+					<h3>g) var x; x == 'undefined'  false.</h3> 
+					<p>It seems like the intent was to initialize a variable but leave it as undefined.  However, variables are already undefined until they are initialzed.  Moreover, the =='undefined' is shown as a string not a value.  Undefined would be a value not a string.</p>  
+
+					<h3>h) '9'<'10' false.</h3>
+					<p>  '' makes these strings not numbers and so they inherently do not have values.</p>
+
+					<h3>i) typeof x + 1 === "number" false. </h3> <p>typeof returns what type of variable x is.  If x is a number, the return would be "number1"-- it concatenates.  If x were a string, the return would be "string1".  Typeof is going to return a description not literally what it is.</p>
+
+					<h3>j) typeof x % 2 === "number" nope.</h3> 
+					<p>Still false.  This isn't the right way to use the typeof operator.  It is used to determine the type of JavaScript variable.</p>
+
+					<h3>k) typeof (x % 2) === "number"  true.</h3><p>Javascript reads from left to right but the paranthesis group x with modulo 2.  Seems that modulo IS a number so even if x were defined as a string and the second value was also a string instead of the number 2, it would still be correct to have the === be "number".</p>
+
+					<h3>l) x++ == ++x false. </h3> 
+					<p>x++ Is typically seen in loops and means to increment.  ++x would mean addition operator and positive number x.  So x++ is typically increment by one, plus positive x would not necessarily be the same thing.</p>
+	
+				</div>
+				<div class="col-md-4">
+					<h3>m) ++x == x++ false if x is a string.  true if x is a number.</h3>
+					<p>  True because ++x is saying "plus positive x" and that is sort of the same as x++ (incrementing).</p>
+
+					<h3>n) "1"+x == 1+x  false. </h3>
+					<p> Javascript sees that 1 is a string and so it's going to want to concatenate.  The answer on the right should thus be 1 concatenated with whatever x was. . . for example, if x = 6, "1" + x would equal "16".  The answer on the right is shown as numbers.</p>
```

Comment:
This is correct in the case that x is a number, but what happens when x is a string?